sdks/java/io/iceberg/build.gradle

```diff
@@ -23,6 +23,8 @@ import java.util.stream.Collectors
 plugins { id 'org.apache.beam.module' }
 applyJavaNature(
         automaticModuleName: 'org.apache.beam.sdk.io.iceberg',
+        shadowClosure: {},
+        validateShadowJar: false,
```

Comment:
Why false? I think we want to validate it. Otherwise we end up having duplicate classes in multiple jars. Basically we want everything in the jar to be in our namespace, not any other namespace.